itest/overcast-docker-itest/src/test/java/com/xebialabs/overcast/host/DockerHostItest.java

```diff
@@ -75,6 +79,39 @@ public void shouldRunMinimalConfig() throws DockerException, InterruptedExceptio
     }
 
     @Test
+    public void shouldRunLinksConfig() throws DockerException, InterruptedException, DockerCertificateException {
+        DockerHost linkedHost = (DockerHost) CloudHostFactory.getCloudHost("mountebankConfig");
+        assertThat(linkedHost, notNullValue());
+        
+        DockerHost itestHost = (DockerHost) CloudHostFactory.getCloudHost("dockerLinksConfig");
+        assertThat(itestHost, notNullValue());
+
+        try {
+            linkedHost.setup();
+            
+            DockerClient dockerClient = new DefaultDockerClient(linkedHost.getUri());
+            String containerId = linkedHost.getDockerDriver().getContainerId();
+            dockerClient.inspectContainer(containerId);
+            
+            itestHost.setup();
+            assertThat(itestHost.getLinks(), contains("mountebank:mountebank"));
+
+            dockerClient = new DefaultDockerClient(itestHost.getUri());
+            containerId = itestHost.getDockerDriver().getContainerId();
+            dockerClient.inspectContainer(containerId);
+
+            try (final LogStream logStream = dockerClient.logs(containerId, LogsParameter.STDOUT, LogsParameter.STDERR, LogsParameter.FOLLOW)) {
+                final String logs = logStream.readFully();
+                assertThat(logs, containsString("Connecting to mountebank:2525"));
+                assertThat(logs, containsString("imposters            100% |*******************************|"));
+            }
```

Comment:
Verifies that the connection to the linked mountebank container has been successfully established